projects/components/src/summary-box/summary-box.component.ts

```diff
@@ -0,0 +1,55 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { Color } from '@hypertrace/common';
+
+@Component({
+  selector: 'ht-summary-box',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `<div class="ht-summary-box">
+    <span
+      class="summary-text"
+      *ngIf="this.count > 0"
+      [htTooltip]="summaryTooltip"
+      [style.color]="this.color"
+      [style.backgroundColor]="this.backgroundColor"
+      [ngClass]="this.displayStyle"
+      >{{ this.summaryText }} {{ this.suffix }}</span
+    >
+    <ng-template #summaryTooltip>
+      <div class="tooltip-contents">{{ this.tooltip }}</div>
+    </ng-template>
+  </div>`,
+  styleUrls: ['./summary-box.component.scss']
+})
+export class SummaryBoxComponent implements OnChanges {
+  @Input()
+  public count!: number;
+
+  @Input()
+  public suffix?: string = 'more';
+
+  @Input()
+  public displayStyle: SummaryBoxDisplay = SummaryBoxDisplay.Plain;
+
+  @Input()
+  public backgroundColor?: Color = Color.White;
+
+  @Input()
+  public color?: string = Color.Gray7;
+
+  @Input()
+  public tooltip!: string;
```

Comment:
We already have a tooltip directive. We can use it directly on the component. No need to add it as another input. If you want, the tooltip can be an internal property for this component.

Reply:
taking string | number | TemplateRef as input and supplying that to htTooltip.